pages/dkp/kommander/2.2/DKP-insight/installation-configuration/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Insight installation and configuration
+title: Insight installation and configuration
+excerpt: Insight installation and configuration
+menuWeight: 30
+beta: true
+enterprise: false
+---
+
+The Insight Management UI is enabled by default with the installation of the DKP platform. No other actions are required.
+
+## Enable insight components
+
+To receive Insights from attached clusters, you must manually enable the Insights components in the attached clusters.
```

Comment:
This part is a bit confusing to me 🤔   Shouldn't insights be enabled by default on clusters? The section above says "No other actions are required.".

Reply:
1. Can you please use "Enable insights engine" instead of "Enable insight components"
2. please use Insight items to represent actual insights in the sentence "To receive Insights from attached clusters"

Reply:
@nataliepina @gvasquezvargas - point 2 above, so I'll go through everything to address this: **Insight item**

Reply:
@

> @nataliepina - This is the first draft of these new pages, so nothing appears in the docs repo until after we merge this PR. The link to the home page for Insights will be: https://docs.d2iq.com/dkp/kommander/2.2/DKP-insights

@nataliepina and @Colin1841 Use the preview:

http://docs-d2iq-com-pr-4225.s3-website-us-west-2.amazonaws.com/